src/onefetch/cli.rs

```diff
@@ -173,6 +183,17 @@ impl Cli {
             None
         };
 
+        let image_colors = if let Some(value) = matches.value_of("image-colors") {
```

Comment:
If you have a `default_value` defined, then the arg should always have *some* value, so it would probably be safe to unwrap `matches.value_of("image-colors")`.